app/responses/admin.py

```diff
@@ -0,0 +1,99 @@
+from django import forms
+from django.contrib import admin
+from django.db.models import Prefetch
+from django.urls import reverse
+from django.utils.html import format_html
+
+from app.prompts.models import Prompt
+from app.responses.models import PromptResponse, Response
+from app.utils.admin import (
+    HideReadOnlyOnCreationAdmin,
+    SetCreatedByOnCreationAdmin,
+)
+from app.utils.views import get_admin_viewname
+
+
+class PromptResponseForm(forms.ModelForm):
+    answer = forms.CharField(required=False)
+
+    class Meta:
+        model = PromptResponse
+        fields = "__all__"
+
+
+class PromptResponseInline(admin.TabularInline):
+    model = PromptResponse
+    form = PromptResponseForm
+    extra = 0
+    can_delete = False
+    ordering = ["prompt__order"]
+    fields = ["prompt_link", "answer", "uuid"]
+    list_display = ["uuid"]
+    readonly_fields = ["uuid", "prompt_link"]
+
+    def prompt_link(self, instance: PromptResponse):
+        """
+        Replaces prompt field in order to include link to the prompt
+        """
+        url = reverse(
+            get_admin_viewname(
```

Comment:
Nice seeing this in utils!

**FYI (no action):**

I don't think it is worth actioning given the shelf life of this code is low. I'd personally consider also:

- using keyword-only arguments (adding `*` in the function signature) as the order of positional arguments is not obvious when you look at the calling code. I do like this idea: https://github.com/octoenergy/public-conventions/blob/7abee9fdd445b062dcc9c054203dc508aafab165/conventions/application.md#keyword-only-functions
- Wrap the whole reverse thing into another util function.